README.md

```diff
@@ -162,10 +163,6 @@ format you want. Refer to Nvidia's GPU support matrix for more details
    conda install "ffmpeg<8" -c conda-forge
    ```
 
-   If you are building FFmpeg from source you can follow Nvidia's guide to
-   configuring and installing FFmpeg with NVDEC support
-   [here](https://docs.nvidia.com/video-technologies/video-codec-sdk/12.0/ffmpeg-with-nvidia-gpu/index.html).
```

Comment:
Drive-by, I don't want to bother readers about installing FFmpeg from source, they shouldn't need to and it shouldn't be in the "entry point" install instructions anyway.